netmanager/src/views/components/Sites/SiteView.js

```diff
@@ -53,6 +53,7 @@ const SiteForm = ({ site }) => {
   };
 
   const handleSubmit = async () => {
+    if(isEmpty(siteInfo.airqlouds)) { delete siteInfo.airqlouds }
```

Comment:
Thanks @FaithDaka , still curious about why this airqlouds field just shows up.
We shall need to continue examining this issue and ensure that only changed fields are submitted in the request body.
Just noticed that libraries like clean-deep have options where you can configure them to your liking. Like for cases where you do not want to remove objects with empty elements, this can be done from here.

Also, most of the endpoint (backend) validation does not allow empty values when field is provided in request body----as of now. But there is room to "relax" this validation so that the client (frontend) can also be able to sometimes send empty values to clear out some fields. This is currently not possible so removing empty fields from your request body could be a workable solution if we are to consider the current backend validations.

Please see screenshot for further details....
<img width="934" alt="Screenshot 2022-03-11 at 10 47 13" src="https://user-images.githubusercontent.com/1590213/157825206-5c70c4e4-8580-403c-a843-4f7560f09e7f.png">